storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2298,8 +2414,13 @@ static void dict_load_from_spaces_sdi() {
     return (DB_SUCCESS);
   });
 
+  dict_load_tables_from_space_id(dict_sys_t::s_space_id, thd, trx);
```

Comment:
Add a comment here on why we need to load DD tables (SDI from mysql.ibd) early?

Reply:
added